.github/workflows/test.yaml

```diff
@@ -52,12 +52,12 @@ jobs:
         env:
           CHARTS_CHANGED: ${{ steps.list-changed.outputs.charts_changed }}
         run: |
-          helm plugin install https://github.com/quintush/helm-unittest > /dev/null 2>&1
+          helm plugin install https://github.com/helm-unittest/helm-unittest > /dev/null 2>&1
```

Comment:
Hey @tiopramayudi, why was this plugin changed here?

Reply:
yesterday i found that new release of that plugin remove `--helm3` flag that previously used, and moving forward the plugin update will be published in the original repository hence change the url for the plugin

Reply:
Thanks, @tiopramayudi. Both the recent versions of the plugin from the original repo don't support the `matchRegex` assertion well. Not sure what the original motivation was to use the `quintush` version but for this reason, I put it back in my new PR: https://github.com/caraml-dev/helm-charts/pull/232

Reply:
Thanks @krithika369 , sorry for the trouble

Reply:
No worries, Tio. Just wanted to clarify the reason for the change. :) Thanks!